src/services/messages-processor/service.ts

```diff
@@ -235,30 +238,33 @@ export const makeMessagesProcessor = ({
     eventsNumber: number
     messages: ExitMessage[]
   }) => {
-    logger.info('Job started')
+    logger.info('Job started', { OPERATOR_ID: config.OPERATOR_ID })
 
     const toBlock = await executionApi.latestBlockNumber()
     const fromBlock = toBlock - eventsNumber
-    logger.info(`Latest block is ${toBlock}`)
 
+    logger.info(`Latest block is ${toBlock}`)
     logger.info(
       `Fetching events for ${eventsNumber} last blocks (${fromBlock}-${toBlock})`
     )
```

Comment:
```suggestion
    logger.info(`Fetching request events from the Exit Bus`, { eventsNumber, fromBlock, toBlock } )
```

Reply:
Moved this comment to the original PR #36